valpas-web/src/views/hakutilanne/HakutilanneTable.tsx

```diff
@@ -173,6 +186,17 @@ const oppijaToTableData = (basePath: string, organisaatioOid: string) => (
   }))
 }
 
+const perusopetusSuoritettu = (oo: OpiskeluoikeusSuppeatTiedot): Value => {
+  const date = oo.päättymispäivä
+  return date !== undefined && oo.näytettäväPerusopetuksenSuoritus
+    ? {
+        value: date,
```

Comment:
Tästä tuli mieleen vielä pari juttua:

(1) Jos opiskeluoikeus on merkitty johonkin eroamistilaan tulevaisuuden päivämäärälle, se näytetään nyt tässä kentässä käsittääkseni suoritettuna. Suorituskentässä ei saisi näyttää kuin oikeasti valmistunut-tilaan päättyneitä tai tulevaisuudessa päättyviä opiskeluoikeuksia.

(2) Ymmärsin Flowdock-keskustelustanne, että tulevaisuudessa 28 päivän sisällä valmistuvat haluttaisiin kuitenkin näyttää eri tavalla kuin menneisyydessä valmistuneet. Näitä ei nyt erotella tässä koodissa.

Nykyinen data (tarkastelupäivänTila + päättymispäivä) ei käsittääkseni näihin riitä, vaan raportointikannasta pitää raahata myös r_opiskeluoikeus.viimeisin_tila kälille, tai sitten tehdä koko logiikka tämän kentän sisällöstä jos SQL:ssä tai Scalassa ja välittää kälille vaan lopputulos. Nyt vähän harmittaa, kun tarkastelupäivänTilaa toteuttaessani poistin jo kerran datassa mukana tulleen viimeisinTila:n, kun sitä ei sllä hetkellä tarvittu missään, ja tuntui vaan hämäävältä.